CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.15.0
+
+_Jul 28, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Aggregation (#5592) @flaviendelangle
```

Comment:
?
```suggestion
- 🎁 Introduce aggregation (#5592) @flaviendelangle
```

Just the word doesn't bring much clarity. Another option can be `Enable aggregation`.

Reply:
Yes, adding a verb make sens